### PR TITLE
[RLlib]: Fix FQE Policy call

### PR DESCRIPTION
**Pull request by Rohan138**
## Why are these changes needed?

Hotfix to make OPE work for both PolicyV2 and DQNTorchPolicy (Policyv1)

## Checks

- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by kouroshHakha — APPROVED** [review]
LGTM.

**Review by richardliaw — CHANGES_REQUESTED** [review]
what's the reason for not having a unit test here?

File: rllib/offline/estimators/tests/test_ope.py [hunk 1]
```diff
@@ -162,6 +166,44 @@ def test_ope_in_algo(self):
         print(*list(std_est.items()), sep="\n")
         print("\n\n\n")
 
+    def test_fqe_model(self):
+        # Miscellaneous tests for FQETorchModel
```
Comment (kouroshHakha) [inline rllib/offline/estimators/tests/test_ope.py:170]: write a docstring on what this function is testing?

**Review by kouroshHakha — APPROVED** [review]
Tests are added. Reviewed again and approved @richardliaw

**Review by richardliaw — APPROVED** [review]

**Pull request merged**